lib/rules/selector-no-deprecated/README.md

```diff
@@ -0,0 +1,87 @@
+# selector-no-deprecated
+
+Disallow deprecated selectors.
+
+<!-- prettier-ignore -->
+```css
+applet {}
```

Comment:
nitpick

I think that using a well known one which has a recommended fix path for the first example would be better.
e.g.
- replacing `nobr` by `white-space: nowrap;`
- replacing `acronym` by `abbr`

Reply:
Updated